src/index.html

```diff
@@ -8,6 +8,47 @@
     <link rel="stylesheet" href="./style.css">
   </head>
   <body>
-    <h1>Stars</h1>
+    <div class="stars--0">
```

Comment:
You should add a BEM block close to modifier (add to other modifiers as well)
```suggestion
    <div class="stars stars--0">
```